osc-rest-server/src/main/java/org/osc/core/broker/rest/server/api/VirtualizationConnectorApis.java

```diff
@@ -496,17 +492,19 @@ public Response updateSecurityGroupMembers(@Context HttpHeaders headers,
             response = BaseJobResponse.class)
     @ApiResponses(value = { @ApiResponse(code = 200, message = "Successful operation"),
             @ApiResponse(code = 400, message = "In case of any error", response = ErrorCodeDto.class) })
-    @Path("/{vcId}/securityGroups/{sgId}/bindings")
+    @Path("/{vcId}/securityGroups/{sgId}/{sfcId}/bindings")
```

Comment:
create new api /{vcId}/securityGroups/{sgId}/sfc/{sfcId}/bindings

create a request for binding with sfc id,     
```
@ApiModelProperty(required = true)
    private Long virtualSystemId;

    @ApiModelProperty(required = true)
    private String name;

    @ApiModelProperty(required = true)
    private Set<Long> policyIds;
```